flows-advanced/using_state_hooks/run_deployment_on_completion.py

```diff
@@ -0,0 +1,33 @@
+"""This examples uses a State Hook to trigger a deployment when a flow run completes.
+
+In this case, the downstream deployment accepts the upstream flow run's result as a
+parameter, therefore we need result persistence to be enabled on the upstream flow.
+"""
+
+from prefect import flow
+from prefect.deployments import run_deployment
+from prefect.filesystems import GCS
```

Comment:
Is result storage here required since we're passing it in the `run_deployment` call?

Reply:
same as the other